ion-schema/src/model/ranges.rs

```diff
@@ -75,7 +74,9 @@ impl<T: RangeBoundType> IonSchemaRange<T> {
         };
         is_above_min && is_below_max
     }
+}
 
+impl<T> IonSchemaRange<T> {
```

Comment:
🗺️ This bit confused me for a moment when I saw it as a diff on GitHub, so here's an explanation. I moved `fn map_bounds()` to a separate `impl` block that has less type bounds on `T` so that I could use it with `NumericRangeType` in `valid_values.rs`.